pkgs/by-name/qd/qdiskinfo/package.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  stdenv,
+  smartmontools,
+  fetchFromGitHub,
+  cmake,
+  qt6,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "QDiskInfo";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "edisionnano";
+    repo = "QDiskInfo";
+    rev = version;
+    hash = "sha256-YLNpQOmh/exBJDuYRUEvzNv/El4x+PttEoj+aCur9wg=";
+  };
+
+  nativeBuildInputs = [ cmake qt6.wrapQtAppsHook ];
+
+  buildInputs = [ qt6.full qt6.qtbase smartmontools ];
+
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE:STRING=MinSizeRel"
+    "-DQT_VERSION_MAJOR=6"
+  ];
+
+  postInstall = ''
+    wrapProgram $out/bin/QDiskInfo \
+      --prefix PATH : ${smartmontools}/bin
+  '';
+
+  meta = with lib; {
+    description = "CrystalDiskInfo alternative for Linux";
+    homepage = "https://github.com/edisionnano/QDiskInfo";
+    license = licenses.gpl3Plus;
+    maintainers = with maintainers; [ roydubnium ];
+    platforms = platforms.linux;
+    mainProgram = "QDiskInfo";
+  };
```

https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes
```suggestion
  meta = {
    description = "CrystalDiskInfo alternative for Linux";
    homepage = "https://github.com/edisionnano/QDiskInfo";
    license = lib.licenses.gpl3Plus;
    maintainers = with lib.maintainers; [ roydubnium ];
    platforms = lib.platforms.linux;
    mainProgram = "QDiskInfo";
  };
```